packages/components/src/tooltip/test/index.js

```diff
@@ -75,18 +70,23 @@ describe( 'Tooltip', () => {
 				</Tooltip>
 			);
 
-			const button = wrapper.find( 'button' );
-			button.simulate( 'focus', event );
+			const button = screen.getByRole( 'button' );
+			button.focus();
 
-			const popover = wrapper.find( 'ForwardRef(Popover)' );
-			expect( originalFocus ).toHaveBeenCalledWith( event );
+			const popover =
```

Comment:
Instead of querying the popover by class and checking that its lenghth is `1`, we could make sure that the popover content is displayed:

```jsx
			// Tooltip text is not rendered when button isn't focused.
			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();

			button.focus();

			// Tooltip test is rendered when button isn't focused.
			expect( screen.getByText( 'Help text' ) ).toBeInTheDocument();
```

Reply:
Fixed by d6f98a7 👍